src/ComboBox/ComboBox.svelte

```diff
@@ -123,6 +123,14 @@
     highlightedIndex = index;
   }
 
+  function clear() {
```

Comment:
This is nice! I noticed that the ListBoxSelection dispatches a "clear" event when a selection is cleared, should we also dispatch one here as well?

Reply:
```suggestion
  export function clear() {
```

Reply:
Suggestion applied.